Update msrepl-agent-jobs.md (#8367)
MicrosoftDocs/sql-docs: docs/relational-databases/system-tables/msrepl-agent-jobs.md

# MSrepl_agent_jobs (Transact-SQL)
[!INCLUDE [SQL Server](../../includes/applies-to-version/sqlserver.md)]

>   The **MSrepl_agent_jobs** table contains information about Replication agent jobs to sync between primary and secondary replicas when the Distribution database is in AG.

Reviewer: ```suggestion
  The **MSrepl_agent_jobs** table contains information about Replication agent jobs to sync between primary and secondary replicas when the Distribution database is part of an availability group. 
```
